src/lib/backend.ts

```diff
@@ -160,14 +147,16 @@ export class ElectronBackend implements Backend {
    * @inheritDoc
    */
   public async storeContext(context: Context): Promise<void> {
-    this.context.set(context);
+    if (this.context) {
+      this.context.set(context);
```

Comment:
Could you throw a `SentryError` here and in `loadContext` instead? We really don't want to call these in the renderer and don't want to miss it in any case. See `sendNativeCrashes` for an example.

Reply:
Yeah, sure. Wasn't sure whether they got called by `core` in the renderer.